packages/gestalt-codemods/14.29.0/toast-change-color-prop.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Converts
+ *  <Toast color='darkGray' /> to <Button color='white'/>
+ */
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  let localIdentifierName;
+  let fileHasModifications = false;
+
+  src.find(j.ImportDeclaration).forEach((path) => {
+    const decl = path.node;
+    if (decl.source.value !== 'gestalt') {
+      return null;
+    }
+
+    localIdentifierName = decl.specifiers
+      .filter((node) => node.imported.name === 'Toast')
+      .map((node) => node.local.name);
+    return null;
+  });
+
+  if (!localIdentifierName) {
+    return null;
+  }
+
+  const transform = src
+    .find(j.JSXElement)
+    .forEach((jsxElement) => {
+      const { node } = jsxElement;
+
+      if (!localIdentifierName.includes(node.openingElement.name.name)) {
+        return null;
+      }
+
+      const attrs = node.openingElement.attributes;
+
+      if (attrs.some((attr) => attr.type === 'JSXSpreadAttribute')) {
+        throw new Error(
+          `Remove Dynamic Text properties and rerun codemod. Location: ${file.path} @line: ${node.loc.start.line}`
+        );
+      }
+
+      let tempAttr;
+      const newAttrs = attrs
+        .map((attr) => {
+          if (
+            attr?.name?.name &&
+            attr.name.name === 'color' &&
+            attr.value.value === 'darkGray'
```

Comment:
can we do `['darkGray', 'white'].includes(attr.value.value)`

then we don't need 
```
tempAttr = [
              j.jsxAttribute(j.jsxIdentifier('color'), j.literal('white')),
            ]
```
because the default is white, so we don't need the code to be redundant

Reply:
just            ` return null;`

Reply:
Gotcha!